### ISSUE TITLE
Props don't change between 2 Image

### ISSUE THREAD

**Issue by kyunkakata** [darenju/react-native-flip-page#2]
![sc1](https://user-images.githubusercontent.com/32797747/44571325-aa67ac00-a7aa-11e8-8d66-4cf41b28f378.PNG)
I got this error when click on this Image, half of it was changed, but the other wasn't. Can you help me to fix this ?

**Comment by darenju** [comment]
Please show some code.

**Comment by kyunkakata** [comment]
This is the code when I use FlipPage:
`
<FlipPage orientation="horizontal">
            <FlipPagePage>
            <Slider idItem={num} leftResize={0} rightResize={-5} lang={this.state.lang}/>
            </FlipPagePage>
            <FlipPagePage>
            <Slider idItem={num+1} leftResize={0} rightResize={-5} lang={this.state.lang}/>
            </FlipPagePage>
          </FlipPage>
`
And this is <Slider class : 
I render 2 Component with boolean selected state:
`render(
if(this.state.selected){
return (<Image source={require('./app/image1.png')}/>)
}else{
return (<Image source={require('./app/image2.png')})
}
)`
When I set clickable TouchableOpacity for those Image. When i click image, half of them was changed (part of image i click), the other wasn't. If I want to the half of right Image change,I will have to click the right part of Image. Sorry for bad at English
